src/modules/wallet/web3-react/connectors/TrezorConnector/index.ts

```diff
@@ -0,0 +1,125 @@
+import { SupportedChainId } from '@cowprotocol/cow-sdk'
+import { Connector } from '@web3-react/types'
+
+import { RPC_URLS } from 'legacy/constants/networks'
+
+import { getCurrentChainIdFromUrl } from 'utils/getCurrentChainIdFromUrl'
+
+import { TrezorProvider } from './TrezorProvider'
+
+import { getHwAccount } from '../../../api/utils/getHwAccount'
+
+import type transformTypedData from '@trezor/connect-plugin-ethereum'
+import type { TrezorConnect } from '@trezor/connect-web'
+
+const defaultChainId = getCurrentChainIdFromUrl()
+
+const trezorConfig: Parameters<TrezorConnect['init']>[0] = {
+  env: 'web',
+  manifest: {
+    email: 'dev@cow.fi',
+    appUrl: 'https://cow.fi',
+  },
+}
+
+export class TrezorConnector extends Connector {
+  public customProvider?: TrezorProvider
+
+  private currentAccountIndex = 0
+
+  private activatedNetwork: SupportedChainId | null = null
+
+  private trezorConnect: TrezorConnect | null = null
+
+  private cancelActivation: () => void = () => void 0
```

Comment:
This looks confusing 😅 

Do you need to declare the type in this case?

Reply:
Yes, unfortunately TS doesn't allow assigning this prop without type